pySDC/implementations/problem_classes/AllenCahn_2D_FD.py

```diff
@@ -119,39 +119,39 @@ def solve_system(self, rhs, factor, u0, t):
 
         u = self.dtype_u(u0).flatten()
         z = self.dtype_u(self.init, val=0.0).flatten()
-        nu = self.params.nu
-        eps2 = self.params.eps**2
+        nu = self.nu
+        eps2 = self.eps**2
 
-        Id = sp.eye(self.params.nvars[0] * self.params.nvars[1])
+        Id = sp.eye(self.nvars[0] * self.nvars[1])
 
         # start newton iteration
         n = 0
         res = 99
-        while n < self.params.newton_maxiter:
+        while n < self.newton_maxiter:
             # form the function g with g(u) = 0
             g = u - factor * (self.A.dot(u) + 1.0 / eps2 * u * (1.0 - u**nu)) - rhs.flatten()
 
             # if g is close to 0, then we are done
             res = np.linalg.norm(g, np.inf)
 
-            if res < self.params.newton_tol:
+            if res < self.newton_tol:
                 break
 
             # assemble dg
             dg = Id - factor * (self.A + 1.0 / eps2 * sp.diags((1.0 - (nu + 1) * u**nu), offsets=0))
 
             # newton update: u1 = u0 - g/dg
             # u -= spsolve(dg, g)
-            u -= cg(dg, g, x0=z, tol=self.params.lin_tol, atol=0)[0]
+            u -= cg(dg, g, x0=z, tol=self.lin_tol, atol=0)[0]
             # increase iteration count
             n += 1
             # print(n, res)
 
-        # if n == self.params.newton_maxiter:
+        # if n == self.newton_maxiter:
```

Comment:
while we are in the process of cleaning up, I guess we can either remove this or add a flag for this.